src/shared.ts

```diff
@@ -143,16 +143,23 @@ export const ESGlobalKeys = SetCreate([
     'Intl', // Unstable
 ]);
 
-// These are foundational things that should never be wrapped but are equivalent
+// These are intrinsics that can be reached by syntax, and must be mapped between realms.
 // TODO: revisit this list.
-export const ReflectiveIntrinsicObjectNames = [
-    'Object',
-    'Function',
-    'URIError',
-    'TypeError',
-    'SyntaxError',
-    'ReferenceError',
-    'RangeError',
-    'EvalError',
-    'Error',
-];
+export function extractUndeniableIntrinsics(globalObj: typeof globalThis): any[] {
```

Comment:
this method can be used on every newly created global object (per realm) to produce the array of constructors to be mapped, then just loop through them.

Reply:
it seems that this is really about undeniable.